server/src/middleware/auth.js

```diff
@@ -3,15 +3,19 @@ import User from "../models/user";
 import constants from "../constants/token";
 
 const auth = async (req, res, next) => {
-  const token = req.header("Authorization").replace("Bearer ", "");
+  const token = req.cookies.token || "";
   try {
-    const data = jwt.verify(token, constants.JWT_KEY);
-    const user = await User.findOne({ _id: data._id, "tokens.token": token });
+    if (!token) {
+      return res
+        .status(401)
+        .send({ error: "Not authorized to access this resource" });
+    }
+    const data = await jwt.verify(token, constants.JWT_KEY);
+    const user = await User.findOne({ _id: data._id });
     if (!user) {
       throw new Error();
```

Comment:
have message in error, else you'll not know if your custom error is thrown or something else

Reply:
I will add it to booking-api's PR.